DNN Platform/Modules/HTML/Components/WorkflowStateController.cs

```diff
@@ -13,16 +13,31 @@ namespace DotNetNuke.Modules.Html
     public class WorkflowStateController
     {
         private const string WORKFLOWCACHEKEY = "Workflow{0}";
+        private const string WORKFLOWS_CACHE_KEY = "Workflows{0}";

```

Comment:
Our current stylecop rules prevent us from using _ (underscores) for these variables which is making the build fail currently.

Reply:
Right, how did I not notice that...